lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.spec.ts

```diff
@@ -329,6 +330,7 @@ describe('ContentMetadataComponent', () => {
             expect(component.node).toEqual(expectedNode);
             expect(contentMetadataService.openConfirmDialog).toHaveBeenCalledWith({nodeType: 'ft:poppoli'});
             expect(nodesApiService.updateNode).toHaveBeenCalled();
+            discardPeriodicTasks();
```

Comment:
why here u don;t need flush ?

Reply:
it worked without flush there, so I just added the minimum to satisfy karma